GeneralsMD/CMakeLists.txt

```diff
@@ -12,5 +12,17 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 # Add main build targets
 add_subdirectory(Code)
 
+# If we are building on windows for windows, try and get the game install path from the registry.
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows" AND CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
Perhaps make this an alias and also ignore case.

Reply:
CMake should always populate these variables correctly cased?